src/main/java/ama/post/CommentPost.java

```diff
@@ -0,0 +1,36 @@
+package ama.post;
+
+import ama.account.User;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+
+/**
+ * Created by Stephane on 2017-03-19.
+ */
+@Entity
+public class CommentPost extends Post {
+
+    @ManyToOne
+    private final Post context;
+
+    public CommentPost() {this.context = null;}
```

Comment:
Had it as a final variable but going to change it so we don't need to initialize to null anymore.